### PR TITLE
[DOCS] Remove redundant ILM attributes

### PR DESCRIPTION
**Pull request by lcawl**
Fixes https://github.com/elastic/elasticsearch/issues/36797

### CODE REVIEW

File: docs/reference/ilm/apis/get-status.asciidoc [hunk 1]
```diff
@@ -1,24 +1,24 @@
 [role="xpack"]
 [testenv="basic"]
 [[ilm-get-status]]
-=== Get {ILM} Status API
+=== Get {ilm} status API
 ++++
-<titleabbrev>Get {ILM} Status</titleabbrev>
+<titleabbrev>Get {ilm} status</titleabbrev>
```
Comment (jakelandis) [inline docs/reference/ilm/apis/get-status.asciidoc]: I assume the change to use the long name is intentional here (as well as other titleabbrev sections) ?

File: docs/reference/ilm/apis/get-status.asciidoc [hunk 2]
```diff
@@ -1,24 +1,24 @@
 [role="xpack"]
 [testenv="basic"]
 [[ilm-get-status]]
-=== Get {ILM} Status API
+=== Get {ilm} status API
```
Comment (jakelandis) [inline docs/reference/ilm/apis/get-status.asciidoc]: I assume the change to use the long name is intentional here (as well as other header sections) ?

File: docs/reference/ilm/apis/put-lifecycle.asciidoc [hunk 3]
```diff
@@ -35,7 +35,7 @@ include::{docdir}/rest-api/timeoutparms.asciidoc[]
 
 You must have the `manage_ilm` cluster privilege to use this API. You must
 also have the `manage` index privilege on all indices being managed by `policy`.
-All operations executed by {Ilm} for a policy are executed as the user that
+All operations executed by {ilm} for a policy are executed as the user that
```
Comment (jakelandis) [inline docs/reference/ilm/apis/put-lifecycle.asciidoc]: I assume the lowercase here is intentional.  However, I don't see any uses of `ilm-cap`. (outside the scope of this PR...but should that be removed?)

**Review by jakelandis — APPROVED** [review]
A couple minor questions, but LGTM.

File: docs/reference/ilm/apis/put-lifecycle.asciidoc (hunk 3 above)
Comment (lcawl) [reply docs/reference/ilm/apis/put-lifecycle.asciidoc]: Yes, I didn't see why the "I" was capitalized in this case.

File: docs/reference/ilm/apis/get-status.asciidoc (hunk 1 above)
Comment (jakelandis) [reply docs/reference/ilm/apis/get-status.asciidoc]: I don't have an opinion either way, just double checking this was intentionally.

**Pull request merged**